backend/src/httpserver/HTTPServer.md

```diff
@@ -0,0 +1,270 @@
+# HTTP Server
+
+Django is a open-source web framework written in Python. It provides a wide range of built-in features and tools that make it easy to build web applications. Django is used for URL routing and handling of HTTP requests for this application. 
```

Comment:
should we put this file in the `shared` directory? I know that it's only about the backend but i feel like it would be good to keep all the documentation in one place. looks really good though!

Reply:
I think that should be fine. We can go ahead with that if Nelson and Joe are okay with your suggestion.

Reply:
We'll put markdown files in shared going foward!